Resources/config/services.yml

```diff
@@ -22,3 +22,6 @@ services:
             $mailer: '@?mailer'
             $translator: '@?translator'
             $templating: '@?templating.engine.twig'
+
+    Chaplean\Bundle\ApiClientBundle\Command\ChapleanApiLogCleanCommand:
+        autoconfigure: true
```

Comment:
- [ ] Should be the default value for all services! (better to add "false" autoconfigure to the one you want to not autoconfigure)